packages/types/src/index.ts

```diff
@@ -206,6 +246,21 @@ export type TransferIframeParams = Pick<
   version: string;
 };
 
+/**
+ * The serialized configuration sent to the Transfer App as a query string.
+ */
+export type SerializedTransferIframeParams = Record<
+  | "partnerId"
+  | "network"
+  | "walletAddress"
+  | "sourceAmount"
+  | "destinationAsset"
+  | "layoutPosition"
+  | "layoutOffset"
+  | "version",
+  string
+>;
```

Comment:
Is there a reason to have this and the `TransferIframeParams`?

Reply:
Yes, the serialized params represent the key/values that will be in the query string whereas `TransferIframeParams` represent the unpacked struct with nested objects.

Open to alternatives for representing this.